app/Repositories/Eloquents/NotificationEloquentRepository.php

```diff
@@ -118,7 +118,17 @@ public function getNotifications($with = [], $data = [], $dataSelect = ['*'])
                     $record = array_add($record, 'link', null);
                     array_push($allRecords, $record);
                     break;
-
+                
+                case config('model.target_type.book'):
+                    if ($record->target) {
+                        $book = $record->target;
+                        $record = array_add($record, 'message',
+                        __('settings.notifications.create_book') . $book->title);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.CloseBracketLine</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/NotificationEloquentRepository.php</br>**Line**: 126</br>Closing parenthesis of a multi-line function call must be on a line by itself